packages/react-api/src/hooks/useTileFeatures.js

```diff
@@ -157,12 +157,12 @@ export default function useTileFeatures({
 }
 
 // WORKAROUND: To read headers and know if the tile is dropping features.
```

Comment:
What's the condition to finally remove this work-around in the future? Can we better describe it @Josmorsot ?
Ping @felixpalmer if required, to have the next step 100% clear

Reply:
we have no condition to remove this. For now it's the only way to do it.